src/app/shared/components/footer/footer.component.html

```diff
@@ -2,19 +2,16 @@
   <a class="flex items-center">
     <app-icon-logo [size]="23" />
   </a>
-  <span class="text-sm text-white font-bold flex items-center space-x-1">
-    <span>made with</span>
-    <svg
-      xmlns="http://www.w3.org/2000/svg"
-      viewBox="0 0 20 20"
-      fill="white"
-      class="w-5 h-5"
-    >
-      <path
-        d="m9.653 16.915-.005-.003-.019-.01a20.759 20.759 0 0 1-1.162-.682 22.045 22.045 0 0 1-2.582-1.9C4.045 12.733 2 10.352 2 7.5a4.5 4.5 0 0 1 8-2.828A4.5 4.5 0 0 1 18 7.5c0 2.852-2.044 5.233-3.885 6.82a22.049 22.049 0 0 1-3.744 2.582l-.019.01-.005.003h-.002a.739.739 0 0 1-.69.001l-.002-.001Z"
-      />
-    </svg>
-    <span>by</span>
+  <span class="text-sm text-gray-400 font-bold flex items-center space-x-1">
+    <span>made with 💚</span>
```

Comment:
in der Vergangenheit (ich glaube das war beim cutover tool) waren Prettier und Emojis nicht die besten Freunde 😄 aber vielleicht klappts ja hier

Reply:
du könntest auch bei einem hero-icon bleiben https://heroicons.com/solid
und fill auf #7aab8a setzen

Reply:
nix, kann ich auch gerne nochmal einsetzen. Fand es so auch ganz hübsch. Ich versuch mal dein Vorschlag